.pronto.yml

```diff
@@ -3,5 +3,3 @@ markdownlint:
     - 'README.md'
     - 'docs/local-development.md'
     - 'jekyll/_cci2_ja/**/*'
-    - 'jekyll/_cci1/**/*'
-    - 'jekyll/_ccie/**/*'
```

Comment:
What is `pronto` used for?

Reply:
it seems like what it is doing is ignoring liniting on certain files I removed it because it didn't need to be excluded anymore since that file location is no longer valid

Reply:
> What is `pronto` used for?

Pronto automates a bot that makes pull request comments - mostly, those that are results of faililng markdown-linting.